docs/homepage.md

```diff
@@ -0,0 +1,146 @@
+page.
+
++ The first entry (designated with the lowest number, usually 1) will display in the top of the main column.
++ The second entry will display below the first entry.
++ The third entry will display in the top of the right sidebar.
++ The fourth entry will display below the third entry.
+
+Note: If multiple entries are given the same order, it cannot be guaranteed that they will display consistently.
+It's better to use 1, 2, 3, and 4 to ensure that the exact order of entries is consistent.
+
+Note: If more than 4 entries are included in this collection, any beyond the fourth entry will be ignored and will not
+display on the homepage.
+
+## Homepage construction
+
+This section is only relevant if significant changes need to be made to the homepage, or if you want to build another 
+page using a similar structure. None of the below information is needed for normal use or maintenance of the site.
+
+### Collections
+
+The homepage takes advantage of the `EntryCollection` model. (You will not have to make any changes to the 
+`EntryCollection` model, but if you want to see how it is constructed, it is found in /writings/models.py) This is 
+already connected to the admin, so the articles included in the collection can be selected and re-ordered there. 
+
+If you are creating a new collection, that will be done in the admin. From the admin homepage, click "Writings", 
+then "Collections", then "Add entry collection". Provide a name (what website admins will see as the name of the 
+collection) and a slug (usually the name, but all lowercase and using '-' instead of spaces. Make sure there are no
+spaces or special characters in the slug). Finally, add a reference to the collection in /newsilkroad/settings.py.
+You'll see a parameter called `CONSTANCE_CONFIG`. Add a new entry, following the pattern of the others:
+
+```
+'MY_NEW_COLLECTION': (
+    'slug-for-my-collection',
+    'Description of the collection', str
+),
+```
+
+### FeaturedEntryMixin
+In order to make sure you have access to the functions which make the articles in the collection available to the 
+template, you will need to subclass the `FeaturedEntryMixin`. You will also need to specify which `EntryCollection` will
+be used to generate the content for this site. That's where you'll reference the collections entry in `CONSTANCE_CONFIG` 
+in /newsilkroad/settings.
+
+The final result will look something like this:
+
+```
+from django.views.generic import TemplateView
+from writings.views import FeaturedEntryMixin
+
+class MyNewPage(FeaturedEntryMixin, TemplateView):
+    template_name = "website/my_template.html"
+    featured_config_key = 'MY_NEW_COLLECTION'
+```
+
+That will point the `FeaturedEntryMixin` to the correct collection, and include the entries in the context.
+
+### Use in Template
+
+The resulting entries will be available in the template as `featured_entry_set`.
```

Comment:
There is also a reference to the first item in this collection via `featured_entry`.